superset/viz.py

```diff
@@ -517,11 +519,11 @@ def json_dumps(self, obj, sort_keys=False):
             obj, default=utils.json_int_dttm_ser, ignore_nan=True, sort_keys=sort_keys
         )
 
-    def payload_json_and_has_error(self, payload):
+    def payload_json_and_has_error(self, payload: VizPayload) -> Tuple[str, bool]:
         has_error = (
             payload.get("status") == utils.QueryStatus.FAILED
             or payload.get("error") is not None
-            or len(payload.get("errors")) > 0
+            or len(payload.get("errors", [])) > 0
```

Comment:
```suggestion
            or bool(payload.get("errors"))
```

Reply:
Good point. Actually `payload.get("errors")` should be suffice.